src/promptflow/promptflow/_cli/_params.py

```diff
@@ -322,11 +322,11 @@ def add_param_output_format(parser):
     )
 
 
-def add_param_owned_only(parser):
+def add_param_include_others(parser):
     parser.add_argument(
-        "--owned-only",
+        "--include-others",
```

Comment:
we could ask long on his opinion on this

Reply:
Synced with long.